ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -473,6 +473,7 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
     var showSearchProgressSpinner: Bool = true
     var showRainbowRingForAvatar: Bool = false
     var showBadgeOnBarButtonItem: Bool = false
+	var showRedDotOnBarButtonItem: Bool = false
```

Comment:
Many of your changes are not properly aligned. This might be caused by using tabs instead of spaces on Xcode.

Reply:
yes updated it